types/socketcluster-server/socketcluster-server-tests.ts

```diff
@@ -81,3 +84,31 @@ agServer.setMiddleware(agServer.MIDDLEWARE_INBOUND, async middlewareStream => {
         action.allow();
     }
 });
+
+// Various server options
+
+agServer = socketClusterServer.attach(httpServer, {
+    wsEngine: require('ws')
+});
+
+agServer = socketClusterServer.attach(httpServer, {
+    wsEngine: 'ws'
+});
+
+agServer = socketClusterServer.attach(httpServer, {
+    authEngine: new agAuth()
```

Comment:
so, if we use proper naming:
```suggestion
    authEngine: new AuthEngine()
```
https://github.com/SocketCluster/socketcluster-server/blob/ea2175a4d8548ca14bd9b5e7ad2cd3e94d70bc40/server.js#L135